lib/index.js

```diff
@@ -40,7 +40,9 @@ exports.register = (server, options, next) => {
   });
 
   server.ext('onPreResponse', (request, reply) => {
-    const rate = request.plugins['hapi-rate-limit'].rate;
+
+    /* istanbul ignore next */
```

Comment:
Yes, but I realized that scenario involved authentication, and that would actually fall out of a reasonable scope for this plugin - it's only sufficient that we make sure the code doesn't break, which this check does. But if you think that we should test this too then I can do that.